app/assess/routes.py

```diff
@@ -29,7 +29,6 @@
 from flask import url_for
 from fsd_utils.authentication.decorators import login_required
 
-
```

Comment:
Looks like just a whitespace change, can we revert?

Reply:
Good point I've re-added in https://github.com/communitiesuk/funding-service-design-assessment/pull/175/commits/e7c94a919f3b27c9a9e796b10a25e0db4cc8c51c, and will squash when merging